.pre-commit-config.yaml

```diff
@@ -26,5 +26,11 @@ repos:
     -   id: mypy
         additional_dependencies: [types-PyYAML, types-requests, types-pkg-resources, types-setuptools]
 
+-   repo: https://github.com/executablebooks/mdformat
+    rev: 0.7.14
+    hooks:
+    -   id: mdformat
+        additional_dependencies: [mdformat-gfm, mdformat-frontmatter, mdformat-footnote]
```

Comment:
What are these extras doing?

Reply:
This is because we have some files, like the issue templates, that are a different flavor or markdown.

The alternative is avoid formatting them but that was getting a little messy.

By installing these plugins, we can support the weird GitHub style markdown and not have things get all messed up.

Reply:
All three of them? or just `mdformat-gfm`?

Reply:
turns out footnote was not needed but the other two were!